.circleci/config.yml

```diff
@@ -107,3 +126,26 @@ workflows:
           requires:
             - build
 
+  deploy_latest_tag:
+    jobs:
+      - deploy:
+          matrix:
+            parameters:
+              tag_name: ["latest"]
+              target: ["arm32v7"]
```

Comment:
I hope they support ARMV6 as well for RaspberryPiZero.

Reply:
@eldesh done by 309b34c

(I don't have a Raspberry Pi Zero, so I can't confirm if it works on a Raspberry Pi Zero.)